client/chain-spec/src/chain_spec.rs

```diff
@@ -99,7 +106,7 @@ enum Genesis<G> {
 	Runtime(G),
 	Raw(
 		HashMap<StorageKey, StorageData>,
-		HashMap<StorageKey, HashMap<StorageKey, StorageData>>,
+		HashMap<StorageKey, (HashMap<StorageKey, StorageData>, Vec<u8>, u32)>,
```

Comment:
maybe add some doc about what is supposed to be Vec<u8> and u32 here

Reply:
Or better yet, use a struct. Tuples of this size are hard to follow.